web3/contract.py

```diff
@@ -62,6 +61,10 @@
     construct_event_filter_params,
     PastLogFilter,
 )
+from web3.utils.validations import (
```

Comment:
Can you rename the `validations` model to `validation`.  Maintaining consistency with the naming conventions from some of my other repositories.